packages/shared/tailwind.config.js

```diff
@@ -186,24 +201,30 @@ module.exports = {
       gap: {
         unset: 'unset',
       },
-      borderRadius: {
-        2: '0.125rem',
-        3: '0.1875rem',
-        6: '0.375rem',
-        8: '0.5rem',
-        10: '0.625rem',
-        12: '0.75rem',
-        14: '0.875rem',
-        16: '1rem',
-        18: '1.125rem',
-        22: '1.375rem',
-        24: '1.5rem',
-        26: '1.625rem',
-        32: '2rem',
-        40: '2.5rem',
-        48: '3rem',
-        50: '3.125rem',
-      },
+      borderRadius: removeDefaults(
+        defaultTheme.borderRadius,
+        ['full', 'none'],
+        {
+          half: '50%',
```

Comment:
Probably better to stick to 1/2 naming as it's common for tailwind

Reply:
Yep. Agree with that.